librz/core/cmd/cmd_analysis.c

```diff
@@ -2177,6 +2177,7 @@ RZ_IPI RzCmdStatus rz_analysis_function_setbits_handler(RzCore *core, int argc,
 	int bits = atoi(argv[1]);
 	RzAnalysisFunction *fcn = analysis_get_function_in(core->analysis, core->offset);
 	if (!fcn) {
+		RZ_LOG_ERROR("core: Cannot find function at 0x%08" PFMT64x "\n", core->offset);
```

Comment:
`analysis_get_function_in` does it for you

Reply:
removed thanks for the info